wpilib/wpilib/watchdog.py

```diff
@@ -0,0 +1,83 @@
+# validated: 2018-09-30 EN a818c7fd4741 edu/wpi/first/wpilibj/Watchdog.java
+# ----------------------------------------------------------------------------
+# Copyright (c) 2018 FIRST. All Rights Reserved.
+# Open Source Software - may be modified and shared by FRC teams. The code
+# must be accompanied by the FIRST BSD license file in the root directory of
+# the project.
+# ----------------------------------------------------------------------------
+
+import wpilib
```

Comment:
Erm.

Reply:
the usual way seemed to preclude mocking notifier - by the time you call patch, the original value is already in watchdog's namespace

Reply:
Not necessarily.

You might also want to consider mocking the Notifier methods instead of the entire class.